unstructured_ingest/v2/processes/connectors/vectara.py

```diff
@@ -0,0 +1,318 @@
+import json
+import typing as t
```

Comment:
Can we use the explicit import rather than importing the entire library with an alias